### DISCUSSION TITLE
Button copy not work when create new repository

### DISCUSSION THREAD

**Discussion by gunzxx** [community/community#54409] [Repositories]
### Select Topic Area

Bug

### Body

I have an issue. I can't copy github link when i create new repository. 
![image](https://user-images.githubusercontent.com/89971946/236107939-3a25e0d7-efb1-45e9-b2c7-4a8f80cfdc3d.png)
Copy button are not working.

**Comment by SamSunshine** [comment]
_The following answer was generated with AI support, hope it helps:_ 

If you're experiencing issues with the copy button not working when trying to copy a GitHub repository link, there are a few troubleshooting steps you can try:

Keyboard shortcut: Instead of using the copy button, you can try using the keyboard shortcut for copying. On most systems, you can press Ctrl+C (Windows/Linux) or Command+C (Mac) after selecting the link to copy it to your clipboard.

Right-click and copy: Another option is to right-click on the link and choose the "Copy" option from the context menu that appears. This should also copy the link to your clipboard.

Browser compatibility: Ensure that you are using an up-to-date and compatible web browser. GitHub is supported on popular browsers like Google Chrome, Mozilla Firefox, Microsoft Edge, and Safari. Try using a different browser or updating your current browser to see if the issue persists.

Clear cache and cookies: Clearing your browser's cache and cookies can sometimes resolve issues with website functionality. Go to your browser settings and look for options to clear browsing data. Make sure to select "Cached images and files" and "Cookies and other site data" before clearing.

Disable browser extensions: Certain browser extensions or add-ons may interfere with the copy button functionality. Try disabling any extensions you have installed, especially those related to clipboard or content blocking, and check if the copy button starts working.

If none of the above steps resolve the issue, it's possible that there might be a temporary bug or issue on GitHub's end. In that case, it would be best to wait for a while and try again later. You can also reach out to GitHub support for further assistance if the problem persists.
